app/views/items/_form.html.erb

```diff
@@ -1,8 +1,8 @@
-<%= simple_form_for(@item) do |f| %>
+<%= simple_form_for (@item), :html => {:multipart => true}  do |f| %>
```

Comment:
I haven't tested this, but the syntax seems problematic.  Could you remove the space after `simple_form_for` and change the params?

I am not familiar with `simple_form_for`.  I apologize if this is how `simple_form_for` works.  If it's not something special about `simple_form_for`, then let me explain where I'm coming from.  I said this on a different PR:

>Ruby can have trouble if there's a space between a method call and the parentheses around its argument(s). Unless you are writing it the exact way you did for a reason, I'd recommend using either parentheses or space but not both. Either of these should work:

```ruby
      <%= simple_form_for @item, :html => {:multipart => true} do |f| %>
      <%= simple_form_for(@item, :html => {:multipart => true}) do |f| %>
```

> (I'm guessing that "@item" and ":html => ..." are both parameters to the simple_form_for method. If so, they should both be inside the parentheses if you use them.)

If this change was made by a generator, let me know what generator -- I am curious!